content/en/deployment/overview.md

```diff
@@ -8,3 +8,115 @@ categories: ["Canary Analysis", "Features", "Concepts"]
 tags: ["Deploy Strategy", "Canary", "Blue/Green", "Kubernetes"]
 ---
 
+
+
+## What is a Deployment
+Deployment is the encompassing construct that delivers your code to remote environments. A deployment
+can deliver software to a single environment or multiple environments either in sequence or in parallel
+depending on your [configuration]({{<ref "deployment/create-deploy-config" >}}).
```

Comment:
```suggestion
depending on your [configuration]({{<ref "deployment/create-deploy-config" >}}).

You can declare CD-as-a-Service deployment configuration as a simple YAML declaratively. The deployment configuration can live inside your source control so that it can be managed as code. You can trigger deployments from your CI system or using the Armory CLI.
```